src/test/skript/tests/syntaxes/functions/factorial.sk

```diff
@@ -0,0 +1,47 @@
+test "factorial numbers":
+    parse:
```

Comment:
you need to actually check the last parse error to see if it failed. this means a different parse section for each value to test.